doc/source/guide/numpy_guide.rst

```diff
@@ -77,35 +77,35 @@ The fact that the ``x.ufunc.negative()`` interface is needed is a known issue wi
 
 NumPy functions as Operators
 ============================
-To solve the above issue, it is often useful to write an `Operator` wrapping NumPy functions, thus allowing full access to the ODL ecosystem. To wrap the convolution operation, you could write a new class::
-
-   class MyConvolution(odl.Operator):
-       """Operator for convolving with a given vector."""
-
-       def __init__(self, vector):
-           """Initialize the convolution."""
-           self.vector = vector
-
-           # Initialize operator base class.
-           # This operator maps from the space of vector to the same space and is linear
-           odl.Operator.__init__(self, domain=vector.space, range=vector.space,
-                                 linear=True)
-
-       def _call(self, x):
-           # The output of an Operator is automatically cast to an ODL vector
-           return np.convolve(x, self.vector, mode='same')
+To solve the above issue, it is often useful to write an `Operator` wrapping NumPy functions, thus allowing full access to the ODL ecosystem. To wrap the convolution operation, you could write a new class:
+
+   >>> class MyConvolution(odl.Operator):
+   ...     """Operator for convolving with a given vector."""
+   ...
+   ...     def __init__(self, vector):
+   ...         """Initialize the convolution."""
+   ...         self.vector = vector
+   ...
+   ...         # Initialize operator base class.
+   ...         # This operator maps from the space of vector to the same space and is linear
+   ...         odl.Operator.__init__(self, domain=vector.space, range=vector.space,
+   ...                               linear=True)
+   ...
+   ...     def _call(self, x):
+   ...         # The output of an Operator is automatically cast to an ODL vector
+   ...         return np.convolve(x, self.vector, mode='same')
```

Comment:
This is only a small example, I'll keep `np.convolve`. Mostly because otherwise the results below are messed up, e.g.
```
File "/home/hkohr/git/odl/doc/source/guide/numpy_guide.rst", line 114, in numpy_guide.rst
Failed example:
    composed_op(x)
Expected:
    26.0
Got:
    26.000000000000004
```